coding-the-legislation/writing_yaml_tests.md

```diff
@@ -165,6 +165,18 @@ To run YAML tests, use the command line tool `openfisca-run-test`, documented [h
 >You can also run tests programatically using the [`test_runner` module](https://openfisca.readthedocs.io/en/latest/test_runner.html).
 
 
+## Testing variables on periods conflicting with their attribute `definition_period`
+
+Is is possible to set a montly input variable for a year using the class attribute `set_input` :
+
+* if `set_input = set_input_divide_by_period`, the 12 months are set equal to the 12th of the input value,
```

Comment:
I'm really not convinced that this belongs here. `set_input` is not really about testing, it is used in all contexts.

And when I'm in the context of writing tests, I may not have the control over my inputs class attributes : if I write an extension for instance, I write tests involving `salary`, but I can't modify its set_input.